include/bitcoin/explorer/commands/qrcode.hpp

```diff
@@ -125,7 +127,8 @@ class BCX_API qrcode
     virtual arguments_metadata& load_arguments()
     {
         return get_argument_metadata()
-            .add("PAYMENT_ADDRESS", 1);
+            .add("PAYMENT_ADDRESS", 1)
+            .add("filename", 1);
```

Comment:
Note the attribute `output="raw"` in the `model/generate.xml`:
```xml
<command symbol="qrcode" category="WALLET" output="raw" stub=...>
```
This attribute does not currently generate code, although that is intended as future work. But it implies that the intended output is bytes. In other word it's intended to be piped output.

The BX command set is pretty consistent with its use of piping. The only commands that use paths are those pertaining to certificate generation. This is because of a present limitation in the ZerMQ implementation, which I have [documented](https://github.com/libbitcoin/libbitcoin-explorer/issues/235) to fix.

We don't want people to script a dependency on cosmetic output, as this creates a fragile and language-specific interface. Notice that error messages are sent to the error stream and the command return code is consistently varied in order to allow users to programatically test command results. These should eventually be options (more below) so it would be redundant to emit them anyway:
```
out2.png: PNG image data, 264 x 264, 1-bit colormap, non-interlaced
```